docs-site/content/28.0/api/search.md

```diff
@@ -173,6 +173,10 @@ curl -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}" \
   </template>
 </Tabs>
 
+:::tip
+Typesense uses offset-based pagination. You can control pagination using the `page` and `per_page` parameters. For more details, check the [pagination parameter documentation](#pagination-parameters).
```

Comment:
Typesense uses pagination by default, with the default page size set to `10`. 

You can control pagination using the `page` and `per_page` parameters, or use offset-based pagination using the `offset` and `limit` parameters. Read more about pagination in the section about [pagination parameter documentation](#pagination-parameters).